src/dfmt/formatter.d

```diff
@@ -1,21 +1,25 @@
-
 //          Copyright Brian Schott 2015.
 // Distributed under the Boost Software License, Version 1.0.
 //    (See accompanying file LICENSE.txt or copy at
 //          http://www.boost.org/LICENSE_1_0.txt)
 
 module dfmt.formatter;
 
-import dparse.lexer;
-import dparse.parser;
-import dparse.rollback_allocator;
-import dfmt.ast_info;
```

Comment:
keep libdparse around for now, abstract your additions into some kind of API.

Reply:
You mean libdparse and dmd should be a switchable flag in dfmt?

Reply:
Yes, both for redundancy and because it will make your diff smaller. It's much easier to merge when you are adding rather than replacing.